tabular/src/autogluon/tabular/models/vowpalwabbit/vowpalwabbit_model.py

```diff
@@ -94,45 +76,35 @@ def _fit(self,
         final_training_data = y + X_series
         final_training_data = final_training_data.tolist()
 
+        extra_params = {
+            'cache_file': 'train.cache',
+            'holdout_off': True,
+            'quiet': True,
```

Comment:
`verbosity` is an extra argument to `fit`. You could set `quiet=False if verbosity > 2` (2 is default, 3 is verbose).

Refer to LightGBM `fit` for an example.

Reply:
Addressed in killerSwitch#1